core/src/components/input/input.tsx

```diff
@@ -272,6 +273,29 @@ export class Input implements ComponentInterface {
    */
   @Prop({ mutable: true }) value?: string | number | null = '';
 
+  /**
+   * Mask format. Can be either a string or array of characters and regular expressions.
```

Comment:
What's the difference between the "string" and "character" terminology?

Reply:
I believe this is answered based on our Slack thread + design doc adjustment proposal, but let me know if that isn't the case. 

Once I have approval on the design doc adjustment, I'll update the type and description for this.